packages/clay-icon/docs/index.js

```diff
@@ -18,11 +17,28 @@ const IconCode = `const Component = () => {
 
 render(<Component />)`;
 
+const iconJSPImportsCode = `<%@ taglib uri="http://liferay.com/tld/clay" prefix="clay" %>`;
+
+const IconJSPCode = `<clay:icon symbol="heart" />`;
```

Comment:
I know you've been following a pattern in this and the other examples, but it's non-obvious to me why we've gone with an initial capital letter for this variable. We usually use leading capitals for things like classes (or factory functions), and components etc. But here we're using it for a local variable that holds a string containing a code snippet (note the inconsistency with `iconJSPImportsCode`, which *also* is a local variable holding a string containing a code snippet).

Reply:
100% it's because that's how other variable names are. It does feel weird having them capitalized, but I went for being consistently inconsistent instead of changing lines that aren't a part of the issue.